### PR TITLE
Change refs-load to refs-restore in refs-restore example

### PR DESCRIPTION
**Pull request by guy-har**

### CODE REVIEW

File: pkg/api/controller.go [hunk 1]
```diff
@@ -2127,8 +2127,11 @@ func (c *Controller) RestoreRefs(w http.ResponseWriter, r *http.Request, body Re
 	}
 
 	// ensure no refs currently found
-	_, _, err = c.Catalog.ListCommits(ctx, repo.Name, repo.DefaultBranch, "", 1)
-	if !errors.Is(err, graveler.ErrNotFound) {
+	commits, _, err := c.Catalog.ListCommits(ctx, repo.Name, repo.DefaultBranch, "", 2)
+	if handleAPIError(w, err) {
+		return
+	}
+	if len(commits) > 1 {
```
Comment (johnnyaug) [inline pkg/api/controller.go:2134]: Why would a bare repository have a commit?

Comment (guy-har) [reply pkg/api/controller.go:2134]: @johnnyaug, The `repository created` commit

Comment (johnnyaug) [reply pkg/api/controller.go:2134]: Maybe it's not documented well enough, but refs-restore should be used with _bare_ repositories, which are create using `lakectl repo create-bare`. This way you restore the repository to the exact state it was, including commit IDs.

Comment (guy-har) [reply pkg/api/controller.go:2134]: I see.
It is documented, I missed that, closing this PR. 
Thanks

**Pull request merged**